cmd/xl-v1-healing.go

```diff
@@ -304,10 +304,7 @@ func healObject(ctx context.Context, storageDisks []StorageAPI, bucket string, o
 	latestDisks, modTime := listOnlineDisks(storageDisks, partsMetadata, errs)
 
 	// List of disks having all parts as per latest xl.json.
-	availableDisks, dataErrs, aErr := disksWithAllParts(ctx, latestDisks, partsMetadata, errs, bucket, object)
-	if aErr != nil {
-		return result, toObjectErr(aErr, bucket, object)
-	}
+	availableDisks, dataErrs := disksWithAllParts(ctx, latestDisks, partsMetadata, errs, bucket, object)
```

Comment:
It seems that there is a little behavior change here and we are now allowing healing an object when not all disks are online, errDiskNotFound was not ignored before. Are we okay with that @harshavardhana ?

Reply:
Yes that is expected, its also fixing a bug in hindsight.